src/components/DeleteSessionDialog.vue

```diff
@@ -7,8 +7,15 @@ const props = defineProps([ 'modelValue', 'deleteId'])
 const emit = defineEmits(['update:modelValue', 'reloadSession'])
 const store = useStore()
 
+const authHeaders = {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json',
+    'Authorization': 'Token 123456789abcdefg',
```

Comment:
Shouldn't this get the authtoken from the store now?

Reply:
noted, used the one from the store now!